conf/illumina.config

```diff
@@ -48,6 +48,8 @@ params {
     // iVar minimum mapQ to call variant (ivar variants: -q)
     ivarMinVariantQuality = 20
 
+    // Run ampliconstats (samtools development branch)
```

Comment:
This this is merged when ampliconstats is in release branch, don't need the " (samtools development branch)" comment anymore.